docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,69 @@ Example payload:
   ]
 }
 ```
+
+### Bulk Document Download URLs
+
+If you want to download all of a patient's documents, especially when they have a large volume of documents, you can start a [Bulk Get Document URL query](/medical-api/api-reference/document/bulk-get-document-url). We’ll send a webhook message with all the downloadable URLs for all the patient's documents.
+
+Here is an example payload:
+
+```json
+{
+"meta": {
+    "messageId": "bd4184dd-3b35-4c60-8130-447d1d528fea",
+    "when": "2023-11-30T05:05:12.215Z",
+    "type": "medical.document-bulk-download"
+  },
+  "patients": [
+    {
+      "status": "completed",
+      "patientId": "<patient-id-1>",
+      "externalId": "<external-id-1>",
+      "documents": [
+        {
+          "id": "018c1b94-57dc-7950-b564-f74ab9b56cc4",
+          "size": 40670,
+          "fileName": "81f2ec4e-c71c-4090-bf1c-9d8529f25b5a/018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16/81f2ec4e-c71c-4090-bf1c-9d8529f65b5a_018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16_018c1b94-57dc-7950-b564-f74ab9b56cc4.xml",
+          "description": "Patient Diagnoses",
+          "status": "current",
+          "indexed": "2019-09-07T15:50:00.000Z",
+          "mimeType": "application/xml",
+          "size": 40670,
+          "type": {
+            "coding": [
+              {
+                "system": "http://snomed.info/sct",
+                "code": "62479008",
+                "display": "Diagnoses"
+              }
+            ],
+            "text": "Diagnoses"
+          }
+          "url": "<a-document-url>"
+        },
+        {
+          "id": "018c1b94-57dc-7950-b564-f749bcf9be3e",
+          "fileName": "81f2ec4e-c71c-4090-bf1c-9d8529f25b5a/018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16/81f2ec4e-c71c-4090-bf1c-9d8529f65b5a_018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16_018c1b94-57dc-7950-b564-f749bcf9be3e.pdf",
+          "description": "Patient Diagnoses",
+          "status": "current",
+          "indexed": "2019-09-07T15:50:00.000Z",
+          "mimeType": "application/pdf",
+          "size": 77093,
+          "type": {
+            "coding": [
+              {
+                "system": "http://snomed.info/sct",
+                "code": "62479008",
+                "display": "Diagnoses"
+              }
+            ],
+            "text": "Diagnoses"
+          }
+          "url": "<another-document-url>"
+        }
+      ],
+    }
+  ]
+}
+```
```

Comment:
as per https://docs.metriport.com/medical-api/more-info/webhooks#patient-document-data - shouldn't we have a "The format follows:" section with a description of the shape of the payload?